### PR TITLE
Removing Bower

### PR DESCRIPTION
**Pull request by richardhallett**
Removing Bower, Use NPM or assets server.
Part of: datacite/datacite#251

---
This change is [Reviewable](https://reviewable.io/reviews/datacite/volpino/58)

### CODE REVIEW

File: vendor/bower.json [hunk 1]
```diff
@@ -1,12 +0,0 @@
-{
-  "name": "vendor",
-  "dependencies": {
-    "jquery-ujs": "~1.0.3",
-    "jquery-bootpag": "~1.0.7",
-    "lodash": "~3.10.0",
-    "d3": "~3.5.5",
```
Comment (mfenner) [inline vendor/bower.json]: Please take a look of what is loaded via the assets server (at https://github.com/datacite/volpino/blob/master/app/views/layouts/application.html.erb#L17):

https://github.com/datacite/segugio/blob/master/source/javascripts/default.js

Comment (mfenner) [reply vendor/bower.json]: We again might be able to load all assets from assets.datacite.org, maybe we need to add a file there.

Comment (richardhallett) [reply vendor/bower.json]: I've removed some of the dependencies that come in from assets server, I think this probably merits some more discussion at some point, but this is suitable for now.

Comment (mfenner) [reply vendor/bower.json]: Sounds good. Volpino is our only Rails app with frontend code, and that is probably replaces by a Javascript frontend next year.

**Pull request merged**